src/assets/scripts/airport/Route/RouteModel.js

```diff
@@ -0,0 +1,133 @@
+import _uniqueId from 'lodash/uniqueId';
+
+/**
+ * Symbol that divides each route segment
+ *
+ * @property SEGMENT_SEPERATION_SYMBOL
+ * @type {string}
+ * @final
+ */
+const SEGMENT_SEPERATION_SYMBOL = '.';
+
+/**
+ * @class RouteModel
+ */
+export default class RouteModel {
```

Comment:
@erikquinn This class will likely need to be extended to work with the fms, once I get to that part.  Could you explain the route format a little but, or maybe correct me where I'm wrong.

A routes takes the shape `ORIGIN_NAME.PROCEDURE_NAME.DESTINATION_NAME`, but I've seen in to comments (somewhere) references to routes in the shape of `ORIGIN_NAME.PROCEDURE_NAME.DESTINATION_NAME..ORIGIN_NAME.PROCEDURE_NAME.DESTINATION_NAME`.

What is the second shape used for?  Is it actually being used within the app somewhere?  Is this just an extended route, like a star to an airway? or an airway to a star?